packages/server/src/ranks/ranks.controller.ts

```diff
@@ -0,0 +1,21 @@
+import isEthereumAddress from 'validator/lib/isEthereumAddress'
+import { Controller, Get, HttpException, HttpStatus, Param } from '@nestjs/common'
+import { RanksService } from 'src/ranks/ranks.service'
+import { ErrorMessage } from 'src/_constants/errors'
+
+@Controller('ranks')
+export class RanksController {
+  constructor(private readonly ranksService: RanksService) {}
+  @Get('/:address')
+  async rank(@Param('address') address: string) {
+    if (!isEthereumAddress(address)) {
```

Comment:
we should normalize the address – postgres does case-sensitive matching, and ethereum addresses have special casing (used as a checksum mechanism). Ethers.js has a great tool to handle that. See here: https://github.com/dOrgTech/cent-social-index/pull/12/files#diff-0e928539541082977fcf0497245c02648b1d1f742ece6c0650538ab08cefc839R2-R27

Reply:
done